Products/ZenModel/migrate/allowGracefulShutdownForZEP.py

```diff
@@ -15,7 +15,7 @@
 from . import Migrate
 
 log = logging.getLogger("zen.migrate")
-sm.require("1.1.14")
+sm.require("1.1.13")
```

Comment:
Why was this changed?

Reply:
This was to work around the following types of traceback when running tests:

`(zenoss) [zenoss@969edd6d734d ~]$ runtests Products.ZenModel.migrate
Preparing configuration files...
Starting Solr for testing purposes...
SOLR_TEST_PID=52
SOLR_TEST_PORT=8993
Waiting for Solr to start...
Solr is running
Starting runtest.py
==============================

Packages to be tested:
        Products.ZenModel.migrate

==============================
Parsing /opt/zenoss/etc/zope.conf
Traceback (most recent call last):
  File "/opt/zenoss/bin/runtests.py", line 512, in <module>
    main()
  File "/opt/zenoss/bin/runtests.py", line 501, in main
    options.count, not options.no_zenpacks
  File "/opt/zenoss/bin/runtests.py", line 435, in runUnitTests
    runZopeTests(cmdline_args)
  File "/opt/zenoss/bin/runtests.py", line 104, in runZopeTests
    sys.exit(testrunner.run(defaults, options))
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/__init__.py", line 30, in run
    failed = run_internal(defaults, args, script_parts=script_parts)
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/__init__.py", line 43, in run_internal
    runner.run()
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/runner.py", line 136, in run
    feature.global_setup()
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 449, in global_setup
    tests = find_tests(self.runner.options, self.runner.found_suites)
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 135, in find_tests
    for suite in found_suites:
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 145, in find_suites
    for fpath, package in find_test_files(options):
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 202, in find_test_files
    for f, package in find_test_files_(options):
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 230, in find_test_files_
    for (p, package) in test_dirs(options, {}):
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 286, in test_dirs
    p = import_name(p)
  File "/opt/zenoss/lib/python2.7/site-packages/zope/testrunner/find.py", line 366, in import_name
    __import__(name)
  File "/opt/zenoss/Products/ZenModel/migrate/__init__.py", line 26, in <module>
    __import__(module[:-3], locals(), globals())
  File "/opt/zenoss/Products/ZenModel/migrate/allowGracefulShutdownForZEP.py", line 18, in <module>
    sm.require("1.1.14")
  File "/opt/zenoss/lib/python2.7/site-packages/servicemigration/version.py", line 23, in require
    raise ValueError("Serviced migrate API %s incompatible with requested version %s." % (API_VERSION, version))
ValueError: Serviced migrate API 1.1.13
 incompatible with requested version 1.1.14.
Tests finished with exit_code=1
Stopping Solr (up to 180 sec)...
`

Reply:
Should I revert these versions in these modules?

Reply:
Yes.

Reply:
Reverted migrate requires version.  All tests passed using service-migration 1.1.14.

https://gist.github.com/chorejasbob/3afe9bf76d0e767a41bd6119f39358ca